CHANGELOG.md

```diff
@@ -56,6 +57,7 @@ All notable changes to this project will be documented in this file.
 
 - ci: Remove Nexus steps from build, mirror and release workflows ([#1056]).
   Also remove the old release workflow.
+- ubi8-rust-builder ([#1091]).
```

Comment:
suggestion: Be a little more expressive what exactly is removed.

```suggestion
- Remove ubi8-rust-builder image ([#1091]).
```

Reply:
I mean, the heading is "Removed", but yeah, extra words can't hurt.

Reply:
Done in 89ab14e